src/NLog/LayoutRenderers/FuncLayoutRenderer.cs

```diff
@@ -92,9 +104,14 @@ private string GetStringValue(LogEventInfo logEvent)
             return null;
         }
 
-        private object GetValue(LogEventInfo logEvent)
+        /// <summary>
+        /// Get the value for the LogEvent
+        /// </summary>
+        /// <param name="logEvent"></param>
+        /// <returns></returns>
+        protected virtual object GetValue(LogEventInfo logEvent)
```

Comment:
Please rename to `RenderValue`. Since it is a LayoutRenderer.

Reply:
done